src/compositor/windowpixmapitem.cpp

```diff
@@ -27,7 +27,11 @@ class SurfaceTextureState {
 public:
     SurfaceTextureState() : m_texture(0), m_xScale(1), m_yScale(1) {}
     void setTexture(QSGTexture *texture) { m_texture = texture; }
-    QSGTexture *texture() const { return m_texture; }
+    QSGTexture *texture() const { return m_texture; }    
```

Comment:
Ah, so that was what changed on that line.  Damn the viewing angle on this monitor/ambient lighting, I was wondering why it was in the diff but the green: completely invisible, so I assumed there was some event more difficult to see red I missed.